### ISSUE TITLE
Update to 2.1.0

### ISSUE THREAD

**Issue by emeryao** [Emeryao/typed-we-app#8]
ref [here](https://mp.weixin.qq.com/debug/wxadoc/dev/devtools/download.html)

**Comment by John0King** [comment]
@Emeryao  current missing `wx.getLogManager`   and  the interface of `LogManager` .  
I'll make a PR when I had time ,  thare many unnecessary `undefined`  in the callback
 ```ts
{ 
     success?:(res?)=>any  // same as  (res:any | undefined )=>any
}
```
in most case the `res : any | undefined`  the `undefined` is not unnecessary

**Comment by emeryao** [comment]
hi @John0King 
thanks for your comment 

the question mark for the callback function param `res` is to force the code to make sure there is a `res` in the callback especially in ts `strict` mode

in other word you have to check the existence of the `res`  because you can not rely on the callback that it always `returns` a result/respone

**Comment by emeryao** [comment]
and `getLogManager` has been published to `npm` with version `1.7.0-update.6`
